frontend/public/components/pod.jsx

```diff
@@ -139,6 +139,30 @@ const Volume = ({pod, volume}) => {
   </div>;
 };
 
+const ContainerTableHead = () => <div className="row co-m-table-grid__head">
```

Comment:
It's not explicitly necessary, but I think the "do I use this elsewhere" is a bit too low of a bar.  If it's potentially reusable and shows to be some sort of atomic unit, no harm, imho. 

Factoring things now can prevent duplication later, because most other devs won't search the code for factorable components before making new things.  We have absurd amounts of duplication in the old `angular` repo, prob due in part to this behavior pattern.

Reply:
This table only makes sense for pods since it shows running state, which is only available for containers for a specific pod. We can't use this on a replica set or deployment page for instance. I don't see a case for reuse here, so I think it makes sense keeping it here and leaving it internal to the pods page component.

It's more about how tightly coupled things are rather than how many lines are in the file. Moving things to separate files if they're tightly coupled doesn't necessarily make things cleaner and can sometimes obscure the relationship.

In general, we should focus on reducing coupling when we can. If things are loosely-coupled, then it's usually obvious that they go in their own file. (For example, the Conditions component we're adding in #97.)

Reply:
Yeah, I agree. My original question was whether this `ContainerTableHead` should be broken out into it's own function at all since it really just renders some static html. I feel like it could this could just be in the `ContainerTable` component, in line.

Reply:
Yeah, I agree. You can't change the columns without changing both the header and table content. Separating them increases the chance that they'll drift, and I'm not sure it makes things any more readable.